app.py

```diff
@@ -0,0 +1,67 @@
+import subprocess
+import os
+from flask import Flask, jsonify, request, make_response
+from flask_sqlalchemy import SQLAlchemy
+from apscheduler.schedulers.background import BackgroundScheduler
+from secrets import SERVICE_ACCESS_TOKEN, MYSQL
+from datetime import datetime
+from models import Users, db
+
+scheduler = BackgroundScheduler()
+app = Flask(__name__)
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://{}:{}@{}/{}'.format(
+    MYSQL['user'],
+    MYSQL['password'],
+    MYSQL['host'],
+    MYSQL['dbname']
+)
+
+def verifyAddition(netid):
+    execproc = subprocess.Popen([r'powershell.exe',
+    './verify_ad_addition.ps1',
+    netid], stdout=subprocess.PIPE, cwd=os.getcwd())
+    out, err = execproc.communicate()
+    if(out.decode().strip() == "1"):
+        Users.query.filter_by(netid=netid).update(dict(added_to_directory=True))
```

Comment:
Also, I'm actually a little concerned why we need any database connection at all.

Shouldn't the users-service make a request with the appropriate user data to the endpoint, trigger the powershell script, and if it gets a 200 back, then update the boolean? Or get an error back, and forward error back to UI?

There seems to be many downsides with accessing the database in this repo, namely:
1. the model needs to be maintained in user.rb and models.py
2. we are now writing to the database in two ways

Reply:
I was just thinking this as well. Ideally, we shouldn't have the `added_to_directory` field, because making a user paid (i.e. setting `is_member` to true) would imply that they've been added to AD

Reply:
Essentially, what I'm suggesting is that this AD service should not have any knowledge of a database. It should just receive a payload of user information, run the powershell script, and return a success/error.

The user service should decide what to do based on that.

I know we have the background job running in the AD service - again, perhaps that should be shifted to the users service.

I don't know, what do you guys think? @bcongdon @aashishkapur

Reply:
Also, eventually we should restrict access to our MySQL database to just accept local connections on the groot prod server. Right now, it's open to the Illinois intranet (we have a secure root password, but an open MySQL db still makes me a bit nervous)

Reply:
Yeah, I think it makes sense to move as much over to the user service as we can.  I still think having two database fields(`added_to_directory` and `is_member`) is good in the case of any errors from ActiveDirectory

Reply:
@aashishkapur can you make a general description of the architecture of this service?
Like from payment to the end of the pipeline and what logic is in which service?

Reply:
Also for future notice, would prefer to have something like this ^ in the description of all PRs that aren't describable in the PR name